testing/correctness/tests/autoscale_tests/Makefile

```diff
@@ -35,7 +35,7 @@ $(abspath $(lastword $(MAKEFILE_LIST)))_RECURSE_SUBMAKEFILES := false
 
 AUTOSCALE_PATH := $(dir $(abspath $(lastword $(MAKEFILE_LIST))))
 CUSTOM_PATH += :$(EXTERNAL_SENDER_PATH):$(TESTING_ALPHABET_PATH):$(TESTING_ALPHABET27_PATH):$(CLUSTER_SHRINKER_PATH)
-CUSTOM_PYTHONPATH += $(TESTING_ALPHABET_PYTHON_PATH):$(TESTING_ALPHABET_PYTHON27_PATH)
+PYTHONPATH :="$(PYTHONPATH):$(TESTING_ALPHABET_PYTHON_PATH):$(TESTING_ALPHABET_PYTHON27_PATH)"
```

Comment:
This will be broken by changes in `resilience-tests` branch, or vice versa.
There are quite a few changes in there to how paths are set in the Makefiles, but the short of it, you should use targets:

```
autoscale_tests: CUSTOM_PYTHONPATH = ...
```
and probably best to copy over the changes to `rules.mk` from that branch.

The reason being... the order in which path modifications are executed affects the total outcome, so it's useful to only execute these conditionally at the time the test target is run.

For Makefiles with multiple target, they can be executed from a single target, with the custom path and pythonpath settings being set under the group target name:
```
group_target: CUSTOM_PATH = ...
group_target: CUSTOM_PYTHONPATH = ...

group_target:
    target1
    target2
    target3
    ...
target1:
    do bla
target2:
    ...
```